sdk/tracing/build.gradle

```diff
@@ -6,9 +6,9 @@ plugins {
     id "ru.vyarus.animalsniffer"
 }
 
-description = 'OpenTelemetry SDK Tracing'
-ext.moduleName = "io.opentelemetry.sdk.tracing"
-ext.propertiesDir = "build/generated/properties/io/opentelemetry/sdk/tracing"
+description = 'OpenTelemetry SDK Trace'
```

Comment:
`Tracing SDK` seems correct

https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/sdk.md#tracing-sdk

I think a change of description from SDK Tracing to Tracing SDK is a good idea.

Reply:
Yes, this should be "OpenTelemetry Tracing SDK"

Reply:
Reverted. just update module name.